### PR TITLE
Refactor mysql call stored procedure BBE

### PR DESCRIPTION
**Pull request by kalaiyarasiganeshalingam**
## Purpose
$Subject

### CODE REVIEW

File: examples/mysql-call-stored-procedures/mysql_call_stored_procedures.bal [hunk 1]
```diff
@@ -119,68 +58,44 @@ type Student record {
...
-            sql:Error? e = studentStream.forEach(function(Student student) {
-                                io:println("Student details: ", student);
-                           });
-        } else {
-            io:println("Empty result is returned from the `GetStudents`.");
-        }
-        checkpanic retCall.close();
+    io:println("Call stored procedure `InsertStudent` is successful.");
+
+    // Process the returned result stream.
```
Comment (praneesha) [inline examples/mysql-call-stored-procedures/mysql_call_stored_procedures.bal:71]:
```suggestion
    // Processes the returned result stream.
```

**Comment by kalaiyarasiganeshalingam** [comment]
> Do not use separate function, move them all here

If this sample demonstrates one function, we can write under the main method. But this sample has three functions. So, If those move under the main method, that method will become multiple lines coding. Therefore, I thought if dividing those into separate methods, that is more clear.  WDYT?

**Comment by niveathika** [comment]
> > Do not use separate function, move them all here
> 
> If this sample demonstrates one function, we can write under the main method. But this sample has three functions. So, If those move under the main method, that method will become multiple lines coding. Therefore, I thought if dividing those into separate methods, that is more clear. WDYT?

The thing is, we have got feedback on the use of functions it's hard to understand the flow since the main method is the last one in the bal file. Though it's the best programming practice when you read the be, it's hard to understand. How about we combine all 3 scenarios into one procedure call and explain. Or I am +1 to show only one scenario too, the others we can explain in the API docs

File: examples/mysql-call-stored-procedures/afterExample.out [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# To create the `Ballerina` project, execute the `bal run` command below
+# and copy this code to that project
+bal new afterExample
+
+# To define the MySQL database driver JAR as a dependency, change the path of
```
Comment (kalaiyarasiganeshalingam) [inline examples/mysql-call-stored-procedures/afterExample.out:5]: @daneshk @niveathika I have added this info here. Are you accept this?

**Review by praneesha — APPROVED** [review]

**Pull request closed without merging**